README.md

```diff
@@ -166,6 +168,16 @@ Changes to the current production infrastructure can be made by editing the file
 4. Copy the Script ID value
 5. Go to google-scripts/clasp.json and update the value associated with scriptId with what you just copied
 
+## Testing
+
+### Manually testing end-to-end
+
+Due to the interaction between Google services such as Google Forms, Docs and Sheets and our main application, it is important to test the full workflow for the MASH functionality manually. This involves filling out the Google Form and viewing the form response and generated Google Doc within the spreadsheet.
+
+A staging version of the Google Form, Document template, Sheet and Apps script (In-Progress) has been created to allow the ability to test the Google related services manually. The Google Form, Document Template and Sheet are held on the shared Google Drive for the Social Care project and has its own folder called _Referrals-MASH Workflow_. If you do not have access to this Drive you can contact one of the content managers of the Drive for access.
```

Comment:
Is the folder _Referrals-MASH Workflow_ for the staging stuff? I'm wondering if we should add staging or test to the name to avoid confusion.

Reply:
When I created the folder I made it with the mindset that this folder will hold all information of MASH related to our team, inside the folder all the staging related material such as the form and sheet have staging included in the name. If we want to however just make the folder only concern itself with the staging materials then I can easily rename the folder

Reply:
I see, the folder is for the dev/design team right? I'm not sure about the folder structure (i.e if we need subfolders) but as long as there's a clear labelling/separation of concerns, I think that's fine. 
Would this also be the folder for any google docs generated via the staging process?

Reply:
Yeah, I imagined both devs and designers could use the folder if they wished. As of right now there is one subfolder which contains the document template and will also be the folder all the generated docs go into